app/controllers/interview_questions_controller.rb

```diff
@@ -0,0 +1,40 @@
+include ActionView::Helpers::TextHelper
+
+class InterviewQuestionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_interviewer
+  before_action :check_template
+  add_breadcrumb "interviews", :interviews_path
+
+  def edit_multiple
+    @interview = Interview.last
```

Comment:
>>> need only one last index
This doesn't make any sense.
BTW what will happen if 2 interviews will conducted in a same moment?